include/wx/bmpcbox.h

```diff
@@ -59,7 +59,7 @@ class WXDLLIMPEXP_CORE wxBitmapComboBoxBase
 #else // wxBITMAPCOMBOBOX_OWNERDRAWN_BASED
 
     // Returns the image of the item with the given index.
-    virtual wxBitmap GetItemBitmap(unsigned int n) const;
+    virtual wxBitmapBundle GetItemBitmap(unsigned int n) const;
```

Comment:
The return type of this one shouldn't changed, similarly to the accessors in #22032.